File: SortAlgorithms/python/SelectionSort/SelectionSort.py
class SelectionSort:
    def sort(self, A):

        array_length = len(A)

        for i in range(1, array_length):
            max_index = 0
            for j in range(1, array_length - i + 1):
                if A[max_index] < A[j]:
                    max_index = j
            A[max_index], A[array_length - i] = A[array_length - i], A[max_index]
        return A

File: SortAlgorithms/php/SelectionSort/SelectionSort.php
<?php

class SelectionSort implements SortArray
{

    public function sort(array $A): array
    {
        $arrayLength = count($A);

        for ($i = 1; $i < $arrayLength; $i++) {
            $maxIndex = 0;
            for ($j = 1; $j <= $arrayLength - $i; $j++) {
                if ($A[$maxIndex] < $A[$j]) {
                    $maxIndex = $j;
                }
            }

            $temp = $A[$arrayLength - $i];
            $A[$arrayLength - $i] = $A[$maxIndex];
            $A[$maxIndex] = $temp;
        }

        return $A;
    }
}